server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -376,4 +376,24 @@ public void test_write_to_subscribed_table_is_forbidden() throws Exception {
             "The relation \"doc.t1\" doesn't support or allow INSERT operations."
         );
     }
+
+    @Test
+    public void test_write_to_subscribed_table_is_allowed_after_dropping_subscription() throws Exception {
+        executeOnPublisher("CREATE TABLE doc.t1 (id INT) WITH(" +
+            defaultTableSettings() +
+            ")");
+        executeOnPublisher("INSERT INTO doc.t1 (id) VALUES (1), (2)");
+        createPublication("pub1", false, List.of("doc.t1"));
+        createSubscription("sub1", "pub1");
+
+        assertThrowsMatches(
+            () -> executeOnSubscriber("INSERT INTO doc.t1 (id) VALUES(3)"),
+            OperationOnInaccessibleRelationException.class,
+            "The relation \"doc.t1\" doesn't support or allow INSERT operations."
+        );
+
+        executeOnSubscriber("DROP SUBSCRIPTION sub1 ");
+        var response = executeOnSubscriber("INSERT INTO doc.t1 (id) VALUES(3)");
```

Comment:
to be fixed: fails with  Expected valid sequence number for replicated op but was unassigned

Reply:
@seut, could it be that I need some extra configs in test or it points to some logical error in implementation?
I added `schemaInfo.invalidateTableCache` in `TransportDropSubscritionAction` to address that issue (ensure that Operations will be recomputed after DROP SUBSCRIPTION as Schemas listen to cluster changes and in debugger I see that it enters it after DROP and resolves back to ALL) - but apparently I missed something else